k6/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "k6",
+  "maintainer": "support@k6.io",
+  "manifest_version": "1.0.0",
+  "name": "k6",
+  "metric_prefix": "k6.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Collect metrics from k6 to visualize and correlate performance testing metrics in DataDog",
```

Comment:
Short description is 80 characters max :(

Reply:
@hithwen Done at https://github.com/DataDog/integrations-extras/pull/578/commits/e6eba49676c8be4fb1e8e9973940527888c16312